File: LanguAI.Frontend/src/api/models/other-user-view-model.ts
/* tslint:disable */
/* eslint-disable */
import { ImageViewModel } from '../models/image-view-model';
import { LastMessageViewModel } from '../models/last-message-view-model';
export interface OtherUserViewModel {
  lastMessage?: LastMessageViewModel;
  profilePicture?: ImageViewModel;
  userId?: number;
  username?: string | null;
}

File: LanguAI.Backend/LanguAI.Backend/ViewModels/User/OtherUserViewModel.cs
using LanguAI.Backend.ViewModels.Image;
using LanguAI.Backend.ViewModels.Message;

namespace LanguAI.Backend.ViewModels.User;

public class OtherUserViewModel
{
    public int UserId { get; set; }
    public string Username { get; set; }
    public ImageViewModel ProfilePicture { get; set; }
    public LastMessageViewModel LastMessage { get; set; }
}
